vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -137,9 +143,15 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         layer.workspace = marlin_make_workspace(
             layer.output_size_per_partition, device)
 
-        # Act-order not supported in compressed-tensors yet, so set to empty.
-        layer.g_idx = marlin_make_empty_g_idx(device)
-        layer.g_idx_sort_indices = marlin_make_empty_g_idx(device)
+        # Handle sorting for activation reordering if needed.
+        has_g_idx = -1 not in layer.weight_g_idx
```

Comment:
we're just checking if this parameter is defined, correct?
having a clear condition on whether or not act order is used is clearer/easier to understand than this case

Reply:
Can you check the config for this? Ditto for when actually making the parameter, it would be good if we avoided making a parameter if the checkpoint doesn't have it

Reply:
I initially avoided relying on the config in order to easily extend to other column orderings (such as weight/group), but if there's a performance cost/ bad vibes associated allocating the parameter (despite it not being passed to the kernel) then we can hard code config values for now

Reply:
I dont believe I follow - your config file is your source of truth for your quantization and will dictate whether or not you're applying act order for your particular model. What would be hardcoded in this case? You'll toggle the value based on whatever you parse from the config